code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -382,10 +395,9 @@
 	qdel(src)
 
 /datum/rune_spell/communication/Destroy()
-	if (destroying_self)
-		return
 	destroying_self = 1
-	qdel(comms)
+	if (comms)
+		qdel(comms)
 	comms = null
```

Comment:
Why setting them to null if they're `qdel`d previously or never existed

Reply:
I was taught to always set my vars back to null to make sure that references are properly removed.